src/gobupload/storage/handler.py

```diff
@@ -59,12 +68,48 @@ def wrapper_decorator(*args, **kwargs):
     return wrapper_decorator
 
 
+class StreamSession(SessionORM):
+    """Extended session class with streaming functionality."""
+
+    # tweak this variable for batchsize
+    # higher value leads to more memory allocation per cycle
+    # 200 shows stable memory usage
+    YIELD_PER = 200
```

Comment:
Seems very low? What do you consider stable memory usage?

Reply:
Will add a bit more text. A higher value (eg 1000) seems to let the memory grow to much higher values. With stable I mean, constant over time + returning memory when possible.

Reply:
YIELD_PER = 2000 now